index.js

```diff
@@ -5,10 +5,23 @@ const path = require('path');
 const fs = require('fs');
 const hashForDep = require('hash-for-dep');
 const HTMLBarsInlinePrecompilePlugin = require('babel-plugin-htmlbars-inline-precompile');
+const VersionChecker = require('ember-cli-version-checker');
+const SilentError = require('silent-error');
 
 module.exports = {
   name: 'ember-cli-htmlbars-inline-precompile',
 
+  init() {
+    this._super.init.apply(this, arguments);
+
+    let checker = new VersionChecker(this);
+    let hasCorrectBabelVersion = checker.for('ember-cli-babel').gt('6.0.0-alpha.1');
```

Comment:
No, I just picked it as the lower bounds and I couldn't recall if .gte is supported 😜